hasura/migrations/default/1704993161810_create_table_pending_app_metadata/up.sql

```diff
@@ -1,26 +1,28 @@
-CREATE TABLE "public"."pending_app_metadata" (
```

Comment:
let's rename this migration as well

Reply:
should we also drop the migration `1704828243607_alter_table_app_verifications`?

Reply:
Yeah removing that migration will just break the current main branch since it's using those fields now so I was planning do it in a separate PR with the front end updates to use the new metadata table